Project 1 (9.12)/stoper/stoper.js

```diff
@@ -1,5 +1,43 @@
-function Stoper() {
-    
-};
+var seconds = 00;
+var tens =00;
+var appendTens =document.getElementById("tens");
+var appendSeconds =document.getElementById("seconds");
+var buttonStart = document.getElementById("button-start");
+var buttonStop = document.getElementById("button-stop");
+var buttonReset = document.getElementById("button-reset");
+var interval; //to store timer values
+//this function will run when click on start
+ function Stoper(){
+    tens++;
+     if(tens < 9){
+        appendTens.innerHTML = "0" + tens;
+     }
+     if(tens > 9){
+        appendTens.innerHTML =tens;
+     }
+     if(tens > 99){
+        seconds++;
+        appendSeconds.innerHTML = "0" +seconds;
+        tens =0;
+        appendTens.innerHTML = "0" + 0;
+     }
+     if (seconds>9){
+        appendSeconds.innerHTML =seconds;
+     }
+ }
+ buttonStart.onclick =function(){
+    interval =setInterval(Stoper);
```

Comment:
musisz podać interwał z jakim funkcja Stoper ma się odpalać. W tym przypadku jest to 10 ms, więc `setInterval(Stoper, 10)`. Dodatkowo, musisz wyczyścić interwał na górze tej funkcji, bo inaczej będzie się dublował za każdym naciśnięciem przycisku